gtk/tests/builder_handlers.rs

```diff
@@ -0,0 +1,145 @@
+#[derive(glib::Downgrade)]
+pub struct MyWidget(gtk::Widget);
+
+#[gtk3_macros::builder_handlers]
+impl MyWidget {
+    fn handler0(&self) {}
```

Comment:
It would seem also useful to provide a `#[signal(name = "name_of_the_signal_handler_in_the_builder_file")]` kind of attribute so that the names don't necessarily have to match exactly.

Not sure what other attributes could be useful, should check what other languages like Vala and Python are doing here.

Reply:
It is already there. I just was too lazy to create an example.

Reply:
Even better :)